config/knip.ts

```diff
@@ -11,12 +11,23 @@ const config: KnipConfig = {
       ignoreDependencies: ["prettier"],
     },
     "packages/*": { entry, project },
-    "packages/alfa-device": { entry: [...entry, "native.ts!"], project },
+    "packages/alfa-cascade": {
+      entry,
+      project,
+      ignore: ["src/ancestor-filter.ts"],
```

Comment:
Seems to be a problem with knip not detecting that something is used in a test file.
Hopefully will be clearable once we reach a more final step.